src/components/Loading/index.js

```diff
@@ -0,0 +1,5 @@
+const Loading = () => {
+  return <h1>Loading...</h1>;
+};
+
+export default Loading;
```

Comment:
same as above, and also it will make your life much easier if you export the component before you declare it, so you wont have to write an extra line of code